sidechain/consensus/aura/src/test/block_importer_tests.rs

```diff
@@ -206,7 +207,7 @@ fn if_block_author_is_self_remove_tops_from_pool() {
 }
 
 #[test]
-fn if_block_author_is_not_self_do_not_remove_tops() {
+fn if_block_author_is_not_self_remove_tops_from_pool() {
```

Comment:
```suggestion
fn executed_tops_get_removed_from_pool() {
```

Reply:
Yes, I think I can consolidate those 2 unit tests into 2 anyway (and re-name accordingly)